Project2015To2017/Reading/NuSpecReader.cs

```diff
@@ -24,14 +24,17 @@ public PackageConfiguration Read(FileInfo projectFile)
 
 			if (nuspecFiles.Length == 0)
 			{
-				this.logger.LogInformation("No nuspec found, skipping package configuration.");
+				this.logger.LogDebug("No nuspec found, skipping package configuration.");
 				return null;
 			}
 
 			if (nuspecFiles.Length > 1)
 			{
-				this.logger.LogError($@"Could not read from nuspec, multiple nuspecs found: 
-{string.Join(Environment.NewLine, nuspecFiles.Select(x => x.FullName))}.");
+				this.logger.LogError(@"Could not read from nuspec, multiple nuspecs found.");
```

Comment:
I think you've inadvertently changed `:` to `.` before the list

Reply:
I can see the difficulty though, since you are making the others ones for information only

Reply:
No, that was fully intentional. Since following list are printed on `Informational` level, they might be not shown at all. The `:` would look weird and force user to think if they have a bug on their side.